arch/arm/boot/dts/overlays/Makefile

```diff
@@ -111,6 +111,7 @@ dtbo-$(CONFIG_ARCH_BCM2835) += \
 	mpu6050.dtbo \
 	mz61581.dtbo \
 	ov5647.dtbo \
+	pac193x.dtbo \
```

Comment:
Is this line spurious?

Reply:
Yes. Compile testing for https://www.raspberrypi.org/forums/viewtopic.php?f=44&t=243562. Sorry.